sktime/networks/es_rnn.py

```diff
@@ -0,0 +1,368 @@
+                        + (1 - seasonal_coeff_1) * seasonality_1[i]
+                    )
+                    seasonality_2.append(
+                        seasonal_coeff_2 * (x[:, i, :] / new_level * seasonality_1[i])
+                        + (1 - seasonal_coeff_2) * seasonality_2[i]
+                    )
+                    level = new_level.unsqueeze(1)
+                seasonality_1 = torch.stack(seasonality_1, dim=1)
+                seasonality_2 = torch.stack(seasonality_2, dim=1)
+                return (
+                    level,
+                    seasonality_1,
+                    seasonality_2,
+                    x
+                    / (
+                        level
+                        * seasonality_1[:, -seq_length:, :]
+                        * seasonality_2[:, -seq_length:, :]
+                    ),
+                )
+
+            def forward(self, x):
+                """
+                Forward pass through ES-RNN.
+
+                Parameters
+                ----------
+                x : torch.Tensor
+                    Input tensor of shape (batch_size, input_length).
+                """
+                if self.seasonality == "zero":
+                    level, new_x = self._nonseasonal(x)
+                    x_input = self.input_layer(new_x.float())
+                    output, _ = self.lstm(x_input)
+                    output = self.output_layer(output[:, -self.horizon :, :])
+                    output_leveled = (output) * level.unsqueeze(1)
+                    return output_leveled
+
+                elif self.seasonality == "single":
+                    level, seasonality_1, new_x = self._single_seasonal(x)
+                    x_input = self.input_layer(new_x.float())
+                    output, _ = self.lstm(x_input)
+                    output = self.output_layer(output[:, -self.horizon :, :])
+                    output_leveled = (
+                        (output) * level * seasonality_1[:, -self.horizon :, :]
+                    )
+                    return output_leveled
+                else:
+                    level, seasonality_1, seasonality_2, new_x = self._double_seasonal(
+                        x
+                    )
+                    x_input = self.input_layer(new_x.float())
+                    output, _ = self.lstm(x_input)
+                    output = self.output_layer(output[:, -self.horizon :, :])
+                    output_leveled = (
+                        (output)
+                        * level
+                        * seasonality_1[:, -self.horizon :, :]
+                        * seasonality_2[:, -self.horizon :, :]
+                    )
+                    return output_leveled
+
+        self._network_class = _ESRNN
+        self.loss = PinballLoss
+
+    def DefaultLoss(self):
+        """Return the default Pinball Loss."""
+        return self.loss()
+
+    def build_network(self):
```

Comment:
```suggestion
    def _build(self):
```
following the convention of other pytorch models